week-1/exercises/decimal.js

```diff
@@ -7,3 +7,13 @@ const num = 58.4567;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+//My answers:
+const wholeNumberPart = Math.round(num - 2);
```

Comment:
you spotted that the variable on line 1 was initialised incorrectly. 

You should use `Math.floor` here to meet the requirement. `Math.round(5.5)` would return `6`, instead of `5`